docs/src/surfaces/drawer/demo/demoDrawer.jsx

```diff
@@ -0,0 +1,618 @@
+import {
+    Drawer,
+    Icon,
+    Typography,
+} from '@saddlebackchurch/react-cm-ui'; // eslint-disable-line import/no-unresolved
+import _ from 'lodash';
+import { connect } from 'react-redux'; // eslint-disable-line import/no-extraneous-dependencies
+import moment from 'moment-timezone';
+import PropTypes from 'prop-types';
+import withStyles from '@saddlebackchurch/react-cm-ui/styles/withStyles'; // eslint-disable-line import/no-unresolved
+import React from 'react';
+import {
+    backgroundColorAlert,
+    backgroundColorSuccess,
+} from '@saddlebackchurch/react-cm-ui/styles/colorExports'; // eslint-disable-line import/no-unresolved
```

Comment:
Are we still supposed to using these?
When you use `withTheme: true` for `withStyles()` HOC, isn't `theme` available in the props?
(Was gonna talk about `useTheme()` hook, since I'm so used to functional components now! 🤣)

Reply:
I Copied/Paste page `demo.jsx` and implemented the same on `demoDrawer.jsx`.

Reply:
Thank you for catching it, you're absolutely right. Now, using `theme` to pass those colors. Fixed.